src/components/productCard/ProductCard.component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Button from "../button/Button.component";
+import {
+  StyledContainerCard,
+  StyledLabelCategory,
+  StyledLabelName,
+  StyledLabelPrice,
+  StyledWrapperInformation,
+  StyledImage,
+} from "./ProductCard.styles";
+
+// eslint-disable-next-line react/prop-types
+function ProductCard({ name, category, price, image, onClick }) {
+  return (
+    <StyledContainerCard title="product-card">
+      <StyledImage src={image} alt={name} />
+      <StyledWrapperInformation>
+        <StyledLabelName>{name}</StyledLabelName>
+        <StyledLabelCategory>{category}</StyledLabelCategory>
+        <StyledLabelPrice>$ {price}</StyledLabelPrice>
+        <Button
+          label="Add to Card"
+          bgColor="#F8F9FA"
+          color="#ADB5BD"
+          width="160px"
+          height="25px"
+          onClick={onClick}
```

Comment:
Make the whole card have the onClick funtionality to navigate to the product details could be a more comfortable, also this way we leave the button "add to cart" available for it's own functionality.
We can also take advantage of react-routers "Link" to achieved this functionality